packages/make-styles/src/runtime/resolveStyleRules.ts

```diff
@@ -68,7 +69,7 @@ export function resolveStyleRules(
       } else {
         result[key] = [className, css];
       }
-    } else if (isObject(value)) {
+    } else if (isRawStyle(value)) {
```

Comment:
I suggest to rearrange conditions instead:

```tsx
if (value == null) {
  // ...
} else if (typeof value === "string" || typeof value === "number") {
  // ...
} else if (property === "animationName") {
  // ...
} else if (isObject(value)) {
  // ...
}
```

In this case we will not need `isRawStyle()` 😺